### DISCUSSION TITLE
Is there a way to ignore all mouse input?

### DISCUSSION THREAD

**Discussion by trsh** [emilk/egui#3504] [Q&A] [answered]
When I click on the app or move mouse, I can see GPU usage going up, but for my very different case I don't want that.

**Comment by JumpyLionnn** [comment, marked as answer]
When you click the app, move the mouse, or create any interaction, `egui` will update the window and render to the screen. `Egui` uses by default reactive mode. It means that the app will do nothing unless you interact with it. so the GPU usage going up is just a normal render. you can see it in a lot of different programs. 
I don't think you want to ignore the mouse input because then you will have a low amount of ways to interact with the app.
Also, i dont think there is a way to do it without creating a custom integration(Ii might be wrong), for that you can take a look at this https://github.com/emilk/egui/blob/master/crates/egui_glow/examples/pure_glow.rs
